guide/designing-products/personal-finance.md

```diff
@@ -177,10 +178,13 @@ A bitcoiner may want to set up a cold storage multi-signature wallet. In this ty
    layout = "float-left"
 %}
 
-A sinking fund is a way to systematically set money aside for a planned considerable expense, like a new car or a down payment for a new home. The amount may be similar to emergency budgets, but the sinking fund is created for a shorter timeline. There are likely to be multiple deposits and potentially only a single withdrawal or transfer, after which the fund is no longer needed. This use case does not necessarily require a dedicated account. The amount could be covered with a dedicated account in either the wallet for monthly budgeting or savings.
+This is money that one sets aside for a considerable but planned expense, such as a new car or a down payment on a house. The amount might be similar to an emergency budget but on a shorter timeline. There are likely to be multiple deposits and potentially only a single withdrawal, after which the fund is no longer needed.
 
 </div>
 
+Sinking funds do not necessarily require a dedicated account. The amount secured could be deposited into any monthly budgeting or savings wallet.
```

Comment:
```suggestion
Sinking funds do not necessarily require a dedicated wallet. The amount secured could be deposited into any monthly budgeting or savings wallet.
```
Are we using account and wallet correctly here? A sinking fund could be a second account in a wallet, assuming that your wallet allows for multiple accounts (which many applications don't).